test/test_simple_imputer.py

```diff
@@ -808,3 +808,32 @@ def assert_val(col, value):
 
     assert_val('global:final_fc_hidden_units', [1])
     assert_val('global:learning_rate', 0.1)
+
+
+def test_hpo_numeric_best_pick(test_dir, data_frame):
+    feature_col, label_col = "feature", "label"
+
+    df = data_frame(feature_col=feature_col,
```

Comment:
there is some data generator for this test which generates maybe slightly more complicated data for a more realistic test:
https://github.com/awslabs/datawig/blob/master/datawig/utils.py#L311

Reply:
oh great, I didn't know that, I'll update it to use that.

Reply:
Though this test case is merely about asserting the ordering of the models and not necessarily based on which input data the model depends on.